src/main/java/edu/cornell/kfs/concur/service/impl/ConcurEventNotificationServiceImpl.java

```diff
@@ -48,10 +49,17 @@ public void updateConcurEventNotificationFlagsAndValidationMessage(ConcurEventNo
         concurEventNotification.setInProcess(inProcess);
         concurEventNotification.setProcessed(processed);
         concurEventNotification.setValidationResult(validationResult);
-        concurEventNotification.setValidationResultMessage(validationResultMessages);
+        concurEventNotification.setValidationResultMessage(truncateValidationResultMessageToMaximumDatabaseFieldSize(validationResultMessages));
         saveConcurEventNotification(concurEventNotification);
     }
     
+    protected String truncateValidationResultMessageToMaximumDatabaseFieldSize(String validationResultMessages) {
+        if (validationResultMessages != null && validationResultMessages.length() > ConcurConstants.VALIDATION_RESULT_MESSAGE_MAX_LENGTH) {
+            validationResultMessages = validationResultMessages.substring(0, ConcurConstants.VALIDATION_RESULT_MESSAGE_MAX_LENGTH);
+        }
```

Comment:
These constant names may be confusing since they are only being used in event notification validation to size the items for the database and have no meaning for the rest of the Concur processing (request extract, collector, sae) validation messages.